Cannot install

When I clone the repo and do an ```npm install``` it fails.
The first error is:
```
Error: Cannot find module 'lodash/cloneDeep'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/Users/mdentremont/ecology/node_modules/babel-template/lib/index.js:59:42)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

--- a/package.json
+++ b/package.json
@@ -62,6 +62,7 @@
     "css-loader": "~0.9.0",
     "fs": "0.0.2",
     "json-loader": "^0.5.4",
+    "lodash": "^4.13.1",
     "marked": "^0.3.5",
     "radium": "^0.17.1",
     "raw-loader": "^0.5.1",
@@ -93,7 +94,6 @@
     "karma-sauce-launcher": "^0.2.14",
     "karma-spec-reporter": "0.0.20",
     "karma-webpack": "^1.7.0",
-    "lodash": "^3.10.1",
     "mocha": "^2.2.5",
     "opener": "^1.4.1",
     "phantomjs": "^1.9.18",
@@ -102,7 +102,7 @@
     "sinon": "^1.16.1",
     "sinon-chai": "^2.8.0",
     "style-loader": "~0.8.0",
-    "stylus-loader": "^1.4.2",
+    "stylus-loader": "^2.1.1",
     "url-loader": "~0.5.5",
     "webpack-dev-server": "^1.14.0"
   }

